### PR TITLE
Upgrade version of `mx-chain-core-go`

### PR DESCRIPTION
**Pull request by miiu96**
- Changed the version of `mx-chain-core-go`
- PR from `mx-chain-core-go`: https://github.com/multiversx/mx-chain-core-go/pull/161

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -7,7 +7,7 @@ require (
 	github.com/gin-contrib/cors v0.0.0-20190301062745-f9e10995c85a
 	github.com/gin-contrib/pprof v1.4.0
 	github.com/gin-gonic/gin v1.8.1
-	github.com/multiversx/mx-chain-core-go v1.1.30
+	github.com/multiversx/mx-chain-core-go v1.1.32-0.20230131083243-ebccba0c240f
```
Comment (iulianpascalau) [inline go.mod:10]: proper release

Comment (bogdan-rosianu) [reply go.mod:10]: :+1:

Comment (miiu96) [reply go.mod]: added

**Review by bogdan-rosianu — APPROVED** [review]

**Review by ssd04 — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]

**Review by iulianpascalau — COMMENTED** [review]
:+1:

**Pull request merged**